README.md

```diff
@@ -33,13 +44,13 @@ The template files below contain steps to add Terraform Init/Plan/Apply/Destroy
   >'location/product/build repo name/environment/component name/terraform.tfstate'  
   _e.g. 'UK South/cft-platform/azure-platform-terraform/sbox/shutter/terraform.tfstate'_  
 
-### Required terraform folder structure:  
+###Required terraform folder structure:  
 Template requires the below folder structure for the build repository.  
 
     Repo
     ├── components                                         
     │   └── <a> (e.g. network)                             # group of .tf files
-    │   │       └── .terraform-version
+    │   │       └── .terraform-version                     # symlink to .terraform-version file at the root level (for local testing)
```

Comment:
can you add the command as well to make it easier?

Reply:
added.